src/rust/scheduler/scheduler.rs

```diff
@@ -71,9 +71,9 @@ pub struct Scheduler {
     /// Maps between externally meaningful ids and the index of the task in the slab.
     task_ids: Rc<RefCell<HashMap<u64, usize>>>,
     /// Holds the waker bits for controlling task scheduling.
-    pages: Rc<RefCell<Vec<WakerPageRef>>>,
+    page_refs: Rc<RefCell<Vec<WakerPageRef>>>,
```

Comment:
I think it's fine to call this waker_pages. It is a reference but the fact that it is a reference isn't important, they are actually being allocated here.

Reply:
I called it `page_refs` because we have defined separate WakerPage and WakerPageRef structures in code explicitly. They are not implicit refs as part of the language, so wanted to have that distinction. Also, there is functionality in the WakerPageRef's non-trivial `default()` implementation (which we use to allocate the pages here).

So, I thought it might be better to call them what they are. Do let me know if you think otherwise.

Reply:
sure can we call them waker_refs or waker_page_refs then? The waker part is more important than the page part.

Reply:
Yes, that makes sense. I will do the change.